cmd/repo.go

```diff
@@ -283,9 +267,74 @@ func readOrUpdateIndex() repo.Index {
 		art.SetUninstaller(verbose, ercoleConfig.RepoService.DistributedFiles)
 	}
 
+	index = append(index, getArtifactsNotIndexed(index, ercoleConfig.RepoService.DistributedFiles)...)
```

Comment:
Maybe the name getArtifactsNotInUpstreamRepository or getArtifactsNotUpstramed is better than getArtifactsNotIndexed

Reply:
But that's not true: that method check scan filesystem for installed artifacts not in passed index, it doesn't check upstream repo!